pkg/virt-operator/webhooks/kubevirt-update-admitter_test.go

```diff
@@ -305,8 +305,13 @@ var _ = Describe("Validating KubeVirtUpdate Admitter", func() {
 				},
 			}))
 		},
-			Entry("with Passt", deprecation.PasstGate, "Passt network binding will be deprecated next release. Please refer to Kubevirt user guide for alternatives."),
-			Entry("with SRIOVLiveMigrationGate", deprecation.SRIOVLiveMigrationGate, "feature gate SRIOVLiveMigration is deprecated, therefore it can be safely removed and is redundant. For more info, please look at: https://github.com/kubevirt/kubevirt/blob/main/docs/deprecation.md"),
+			Entry("with LiveMigration", deprecation.LiveMigrationGate, fmt.Sprintf(deprecation.WarningPattern, deprecation.LiveMigrationGate, deprecation.GA)),
```

Comment:
I think the test intentional hard-coded the messages, so if the production code change the test will not pass, in this case, the  warning message format.
It enables having having the composed warning messages visible.
It also saves exposing variables just for tests.

Reply:
I'm not sure it's worth asserting against hard coded strings as the feature gate and feature states are the only things that can change here and both are captured with the use of the pattern. Changes to the pattern itself shouldn't warrant changes in the tests IMHO.

Reply:
hmm. I also don't see an issue with the current approach.